README.md

```diff
@@ -175,39 +216,47 @@ The generated Event Management Agent jar file is found in:
 application/target/event-management-agent-1.0.0-SNAPSHOT.jar
 ```
 
-## Deployment
+### Running the Event Management Agent as a process
 
-There are two main modes of deployment:
+You can run the Event Management Agent as a process in both connected and standalone modes.
 
-* Connected: The Event Management Agent connects to the Solace PubSub+ Event Portal. Scans are initiated from Event Portal. Completed scan results are uploaded to Event Portal automatically.
+Specify:
+* the Event Management Agent jar file (e.g. event-management-agent-1.0.0-SNAPSHOT.jar)
+* the location of the connection file (e.g. /path/to/file/AcmeRetail.yml) - See [Generating the Event Management Agent connection file](#generating-the-event-management-agent-connection-file) to generate a connection file
 
-* Standalone: The Event Management Agent is controlled via the REST API and results must be uploaded manually.
+```
+java -jar application/target/event-management-agent-1.0.0-SNAPSHOT.jar --spring.config.location=/path/to/file/AcmeRetail.yml
+```
 
-## Running a scan
+The Event Management Agent is ready.
 
-### REST interface
+# Broker Plugins
 
-The Event Management Agent includes a REST API that allows the user to initiate a scan. Each plugin requires its own
-custom set of authentication and identification attributes that must be supplied by the user.
+The Event Management Agent comes with the following event or message broker plugins included:
 
-See [REST Documentation](docs/rest.md) for additional information
+* Apache Kafka
+* Solace PubSub+
+* Confluent Schema Registry
+* MSK
+
+The default application.yml provides various plugin examples. For Kafka, the properties section under credentials is passthrough. For example a property in ConsumerConfig or SSLConfigs classes.
+
+If using AWS IAM, the AWS Access Key Id and AWS Secret Access Key need to be present. This can be done via a credentials file or environment variables as shown below:
 
-## Importing Scanned Data
+Put a file with the following contents into a ~/.aws/credentials file
+
+```
+[default]
+aws_access_key_id = <aws_access_key>
+aws_secret_access_key = <aws_secret_key>
+```
 
-In you want to import your scanned data into the Event Portal in the standalone mode, you will need to do so manually with the following steps:
+Can alternatively make these environment variables (these will also override the credentials file if present)
```

Comment:
Wording: it sounds more natural to say "Alternatively you can make these ..."

Reply:
Done